analysers/analyser_osmosis_waterway.py

```diff
@@ -149,8 +149,8 @@ def __init__(self, config, logger = None):
 '''There is one `waterway=riverbank` but there is no `waterway=river`
 inside it.'''),
             fix = T_(
-'''After checking, create a "river" line inside te "riverbank"
-polygon.'''))
+'''After checking, create a "river" line inside the "riverbank"
+polygon or eliminate "riverbank" polygon.'''))
```

Comment:
```suggestion
polygon or eliminate then "riverbank" polygon.'''))
```
```suggestion
polygon or eliminate "riverbank" polygon.'''))
```

Reply:
I am unsure what is being proposed here. Adding word "then" and removing it?

Reply:
Ho, it is a typo. I was suggesting adding "the".